server/bundler/plugin.js

```diff
@@ -30,11 +31,12 @@ ChunkFileNames.prototype.apply = function( compiler ) {
 						"script.onerror = script.onload = script.onreadystatechange = null;",
 						"delete installedChunks[ chunkId ];",
 						"window.__chunkErrors[ " + JSON.stringify( chunkMaps.name ) + "[chunkId]||chunkId ]=new Error();",
-						"callback.call( null, " + this.requireFn + ")"
+						"return Promise().resolve();"
```

Comment:
I meant this one. Yeah, it does the retry stuff... Not sure how to replace that. 

https://github.com/Automattic/wp-calypso/pull/10634 teaches webpack to minimize our JS, which has other benefits. We'd lose the ability to serve the unminified version easily, but if we turn on sourcemaps for production-y environments, it's not that much of a loss.